Translated techlab 3.5 to english (#31)
appuio/techlab: labs/04_deploy_dockerimage.md

```
to show all projects the current user is authorized to see.

> Once you crated the new project you can deploy the Docker Image in OpenShift using:

Reviewer: ```suggestion
Once you have created the new project you can deploy the Docker Image using:
```

4. Select Pods

> **Hint** To Create your own Docker Image to run on OpenShift, you should follow these best Practices: https://docs.openshift.com/container-platform/3.5/creating_images/guidelines.html

Reviewer: ```suggestion
**Hint** In order to create your own container image to run on OpenShift, you should follow these best practices: https://docs.openshift.com/container-platform/3.5/creating_images/guidelines.html
```
